openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -664,6 +678,7 @@ def _to_smirnoff_data(self, discard_cosmetic_attributes=True):
 
         smirnoff_dict = OrderedDict()
         smirnoff_dict['SMIRNOFF'] = l1_dict
+        smirnoff_dict = convert_smirnoff_data_quantitys_to_string(smirnoff_dict)
```

Comment:
"quantities" is misspelled.

Reply:
This is a style thing for me -- since the object is `Quantity`, I write the plural  as `Quantity`s in docstrings (some day we can make sure all of these resolve to links to class names). It's not necessary to do this in function names but it seemed to be consistent. Weak preference, happy to change if it's more confusing than helpful.

Reply:
What about `convert_smirnoff_data_quantity_to_string` then?

Reply:
`convert_smirnoff_data_quantity_to_string`. I've changed it to that.